whisper/whisperv6/envelope.go

```diff
@@ -51,6 +51,7 @@ type Envelope struct {
 
 // size returns the size of envelope as it is sent (i.e. public fields only)
 func (e *Envelope) size() int {
+	const EnvelopeHeaderLength = 20
```

Comment:
Why isn't this constant declared in doc.go ?

Reply:
because it is used only once in this function, and never again. we don't need a global constant if it can be local.

Reply:
I disagree: in spite of momentarily only being used in this specific function, this constant is a characteristic of the envelope format, and therefore its scope is the whole envelope code. So it should definitely be a top-level function, if not documented in `doc.go`.

Reply:
To press the point further, there are two reasons this should be defined globally:
  * Anybody who would need it will not be scanning the code to find if it's already defined. They will look at `doc.go` and the first lines of `envelope.go`.
  * documentation purposes: once again, not having to scan the entire code to get some info about the format.